src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+    private static final String ADD_CONFIRMATION_MESSAGE = "Got it. I've added this task:";
+    private static Task[] inputs = new Task[100];
+    private static int index = 0;
+
+    /**
+     * Returns a string with formatted user inputs.
+     *
+     * @param inputs list of user input.
+     * @return Formatted string to print
+     */
+    public static String formatList(Task[] inputs) {
```

Comment:
Overall, your main Duke class is doing a lot of heavy lifting. You may want to consider creating a separate class to deal with formatting and getting user input(UI class), and maybe even one more for just creating and managing Tasks (TaskManager class), and make the Duke class a little cleaner and easier to follow.